.github/workflows/consensuswarn.yml

```diff
@@ -18,4 +18,4 @@ jobs:
       - uses: actions/checkout@v4
       - uses: orijtech/consensuswarn@main
         with:
-          roots: "github.com/evmos/evmos/v19/app.Evmos.DeliverTx,github.com/evmos/evmos/v19/app.Evmos.BeginBlocker,github.com/evmos/evmos/v19/app.Evmos.EndBlocker"
+          roots: "github.com/evmos/evmos/v19/app.Evmos.FinalizeBlock,github.com/evmos/evmos/v19/app.Evmos.BeginBlocker,github.com/evmos/evmos/v19/app.Evmos.EndBlocker"
```

Comment:
I'm wondering if there are any other things from ABCI++ or 2.0 or whatever to be added here? Like the `PrepareProposal`, `ProcessProposal`, etc.?

Reply:
At this moment we are not supporting these methods of ABCI++  from my knowledge. But I can add them so we don't forget to update this action later.

Reply:
I think if we do that then we're close to done with this PR

Reply:
I was thinking that I'm not sure how this CI is using root, so I believe we should first merge with the roots we have in our codebase, and then add others. I don't know if it will continue to complain because ProposalXXXX methods are not implemented. Do you have a better understanding of this action?

Reply:
we just need this and some Python lint and then we're done

Reply:
> Do you have a better understanding of this action?

I'm personally not too sure, as I understand it's a linter catered towards consensus breaking changes in Cosmos SDK based chains and is listening for changes in consensus relevant logic, which you define in the `roots` input? 🤔